spec/fixtures/multiline-soa

```diff
@@ -0,0 +1,7 @@
+osuosl.org      IN SOA  ns1.auth.osuosl.org. hostmaster.osuosl.org. (
```

Comment:
If we can make only the one with hyphens in the ns server name pass, it would be good.

Else we file these both test case as issues and keep them alive and also document them somewhere in the recipe just in case we come back to the recipe in a traceback.